src/Symfony/Component/Security/Http/Tests/Authentication/AuthenticatorManagerTest.php

```diff
@@ -78,9 +84,12 @@ public static function provideSupportsData()
         yield [[], false];
     }
 
+    /**
+     * @group legacy
```

Comment:
We cannot make all tests legacy. when removing deprecations, we remove legacy tests as well.

Tests that are not specifically about ensuring that the BC layer works fine should be migrated to use the non-deprecated API.
Some tests might be duplicated to test the same case with the old and the new configuration way, when it is about covering specifically the feature for which we have a BC layer.

For instance, `supports` is unrelated to the deprecation you introduce, so its tests have no reason to become legacy.

Reply:
I've duplicated the test, so that the normal test is only testing the new logic and the `BCTest` is testing the new behavior.